doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -2902,6 +2902,7 @@ pattern will be used to process its instance.
 
      Axiom P : nat -> Prop.
      Axioms eqn leqn : nat -> nat -> bool.
+     Declare Scope "this_scope".
```

Comment:
Currently stucked there.
Without it:

```
Extension error:
/home/theo/git/coq/_build/default/doc/sphinx/proof-engine/ssreflect-proof-language.rst:2771: Error while sending the following to coqtop:
    Notation "a != b" := (eqn a b) (at level 70) : this_scope.
  coqtop output:
    Toplevel input, characters 0-58:
    > Notation "a != b" := (eqn a b) (at level 70) : this_scope.
    > ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    Error: Declaring a scope implicitly is deprecated; use in advance an explicit
    "Declare Scope this_scope.".
```

With it:

```
Extension error:
/home/theo/git/coq/_build/default/doc/sphinx/proof-engine/ssreflect-proof-language.rst:2771: Error while sending the following to coqtop:
    Declare Scope "this_scope".
  coqtop output:
    Toplevel input, characters 8-13:
    > Declare Scope "this_scope".
    >         ^^^^^
    Error:
    Syntax error: 'Equivalent' 'Keys' expected after 'Declare' (in [vernac:command]).
```

Reply:
OK the trouble was with the quotes. But the syntax error message is really weird.